### PR TITLE
Onboarding: Make log output texts friendlier to new users

### PR DESCRIPTION
**Pull request by practicalswift**
Make log output texts friendlier to new users.

### CODE REVIEW

**Comment by practicalswift** [comment]
@ZmnSCPxj Thanks! Now fixed! :-)

File: lightningd/lightningd.c [hunk 1]
```diff
@@ -332,9 +332,9 @@ int main(int argc, char *argv[])
 	setup_jsonrpc(ld, ld->rpc_filename);
 
 	/* Mark ourselves live. */
-	log_info(ld->log, "Hello world from %s aka %s #%s (version %s)!",
+	log_info(ld->log, "Server started with public key %s, alias %s and lightningd %s",
 		 type_to_string(ltmp, struct pubkey, &ld->id),
-		 ld->alias, tal_hex(ltmp, ld->rgb), version());
+		 ld->alias, version());
```
Comment (rustyrussell) [inline lightningd/lightningd.c]: How about `(color #%s)`? I think color is underused at the moment, but it'll eventually be critical :)

Comment (practicalswift) [reply lightningd/lightningd.c]: Fixed! :-)

Comment (cdecker) [reply lightningd/lightningd.c]: :100:

**Pull request merged**